add help with overpass-turbo query for calculated postcode

mtmail opened the pull request:
fixes https://github.com/osm-search/nominatim-ui/issues/113

lonvia commented:
Nice. Nominatim also accepts the tag `postal_code`. Can that be added to the Overpass-Query.

mtmail commented:
Added `postal_code`. Also switched from 10km radius search to map bbox, that makes it easier for users to zoom out and rerun the query. Uses a higher initial zoom so the query doesn't time out in dense areas.
